src/cli/init/index.ts

```diff
@@ -20,29 +20,38 @@ import {
   tsConfig,
 } from "./templates";
 import { createDir, sanitizeName, write } from "./utils";
-import { confirm, walkthrough } from "./walkthrough";
+import { confirm, PromptOptions, walkthrough } from "./walkthrough";
+import { ErrorList } from "../../lib/errors";
 
 export default function (program: RootCmd) {
+  let response = {} as PromptOptions;
+  let pkgOverride = "";
   program
     .command("init")
     .description("Initialize a new Pepr Module")
-    // skip auto npm install and git init
-    .option("--skip-post-init", "Skip npm install, git init and VSCode launch")
-    .action(async opts => {
-      let pkgOverride = "";
-
-      // Overrides for testing. @todo: don't be so gross with Node CLI testing
+    .option("--confirm", "Skip verification prompt before creating module")
+    .option("--description <string>", "Set a description!")
+    .option("--name <string>", "Set a name!")
+    .option("--skip-post-init", "Skip npm install, git init, and VSCode launch")
+    .option(`--errorBehavior [${ErrorList}]`, "Set a errorBehavior!")
```

Comment:
```suggestion
    .option(`--errorBehavior [${ErrorList}]`, "Set a errorBehavior!")
```
Looking for something more like `--errorBehavior <errorBehavior>`
For Example:
```ts
   new Option(
        "--registry <GitHub|Iron Bank>",
        "Container registry: Choose container registry for deployment manifests. Can't be used with --custom-image.",
      ).choices(["GitHub", "Iron Bank"]),
  ```

The brackets `<>` in commander make it required and this is a required field. Let the user know what the default is. Remove the exclamation point. 

https://github.com/tj/commander.js?tab=readme-ov-file#required-option
https://github.com/defenseunicorns/pepr/blob/7e7b03eab0c1f367be7bd2a484f0e5fec7088fe6/src/cli/build.ts#L62